src/Spd.Resource.Repository/Biz/BizRepository.cs

```diff
@@ -69,10 +69,7 @@ public async Task<IEnumerable<BizResult>> QueryBizAsync(BizsQry qry, Cancellatio
                 throw new ApiException(HttpStatusCode.InternalServerError, $"Biz {biz.name} does not have service type.");
 
             var licenceId = biz.spd_organization_spd_licence_soleproprietor
-                .FirstOrDefault(l => l.statecode == DynamicsConstants.StateCode_Active)?.spd_licenceid;
-
-            var contactId = biz.spd_organization_spd_licence_soleproprietor
-                .FirstOrDefault(l => l.statecode == DynamicsConstants.StateCode_Active)?._spd_licenceholder_value;
+                .FirstOrDefault()?.spd_licenceid;

```

Comment:
I will approve for now. There is problem, we need to talk to Dynamics. We will do it tmr